gulpfile.js

```diff
@@ -4,7 +4,7 @@ var chmod = require('gulp-chmod');
 
 gulp.task('modularize', function() {
   gulp.src('./infobox-lib.js')
-    .pipe(concat.footer('\n\nmodule.exports = { default: InfoBox, InfoBox: InfoBox }; '))
+    .pipe(concat.footer('\n\nexports.default = InfoBox;\nexports.InfoBox = InfoBox;'))
```

Comment:
@mxl I hope changing to two export statements would be okay. (Just want to make it more friendly to bundler).